doc-content/enterprise-only/openshift/template-deploy-ldap-proc.adoc

```diff
@@ -25,8 +25,11 @@ If you want to enable LDAP failover, you can put set or more LDAP server address
 +
 If the LDAP server does not define all the roles required for your deployment, you can map LDAP groups to {PRODUCT} roles. To enable LDAP role mapping, set the following parameters:
 +
-** *RoleMapping rolesProperties file path* (`AUTH_ROLE_MAPPER_ROLES_PROPERTIES`): The fully qualified path name of a file that defines role mapping, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`. You must provide this file and mount it at this path in all applicable deployment configurations; for instructions, see <<rolemapping-proc_{context}>>.
-** *RoleMapping replaceRole property* (`AUTH_ROLE_MAPPER_REPLACE_ROLE`): If set to `true`, mapped roles replace the roles defined on the LDAP server; if set to `false`, both mapped roles and roles defined on the LDAP server are set as user application roles. The default setting is `false`.
+** *RoleMapping rolesProperties file path or one lined roles* (`AUTH_ROLE_MAPPER_ROLES_PROPERTIES`): The fully qualified path name of a file that defines role mapping, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`. You must provide this file and mount it at this path in all applicable deployment configurations; for instructions, see <<rolemapping-proc_{context}>>.
++
+Alternatively, you can enter role mapping settings directly in this property, using the `role=role1,role2;another-role=role2` pattern, for example: `admins=kie-server,rest-all,admin;developers=kie-server,rest-all`.
+** *Role Mapper Keep Mapped* (`AUTH_LDAP_MAPPER_KEEP_MAPPED`): If set to `true`, both mapped roles and roles defined on the LDAP server are set as user application roles; if set to `false`, mapped roles replace the roles defined on the LDAP server. The default setting is `false`.
```

Comment:
will this be added on the operator doc as well?

Reply:
As soon as I work out what it looks like in operator console :)